main-actions/src/main/scala/sbt/RawCompileLike.scala

```diff
@@ -47,16 +47,20 @@ object RawCompileLike {
   def cached(cacheStoreFactory: CacheStoreFactory, doCompile: Gen): Gen =
     cached(cacheStoreFactory, Seq(), doCompile)
 
-  def cached(cacheStoreFactory: CacheStoreFactory,
-             fileInputOpts: Seq[String],
-             doCompile: Gen): Gen =
+  def cached(
+      cacheStoreFactory: CacheStoreFactory,
+      fileInputOpts: Seq[String],
+      doCompile: Gen
+  ): Gen =
     (sources, classpath, outputDirectory, options, maxErrors, log) => {
       type Inputs =
         FilesInfo[HashFileInfo] :+: FilesInfo[ModifiedFileInfo] :+: Seq[File] :+: File :+: Seq[
-          String] :+: Int :+: HNil
+          String
+        ] :+: Int :+: HNil
```

Comment:
A dangling square bracket? Is this more readable, srsly?

Reply:
this one should be rewrapped differently, agreed